app/controllers/concerns/hydranorth/files_controller_behavior.rb

```diff
@@ -12,8 +12,22 @@ module FilesControllerBehavior
 
 
     included do
-      self.edit_form_class = Hydranorth::Forms::GenericFileEditForm
-      self.presenter_class = Hydranorth::GenericFilePresenter
+      def presenter_class
+        super unless @generic_file
+        return Hydranorth::CstrPresenter if cstr_resource?
```

Comment:
Since this depends on the methods in FilesController anyways, why not move it over there? It looks like that's the only place this is mixed in anyways so it shouldn't impact anything else.